packages/dashboard/src/app/views/templateDetails/index.js

```diff
@@ -126,21 +129,23 @@ function TemplateDetails() {
     if (!template || !templateId) {
       return;
     }
-    const id = parseInt(templateId);
+
+    const allRelatedTemplates = getRelatedTemplatesIds(
+      template,
+      templatesByTag
+    );
+
+    const randomRelatedTemplates = allRelatedTemplates
+      .sort(() => 0.5 - Math.random()) // Randomly sort the array of ids;
+      .slice(0, 6); // Get first 6 templates
```

Comment:
Every time the page renders there will be a new set of templates (as long as there are more than 6). They will always be in a random order.

![random-templates](https://user-images.githubusercontent.com/22185279/132104859-1d60191c-51de-47f6-ab78-59ad438d2714.gif)